artemis/modules/base/base_newer_version_comparer.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+import subprocess
+import tempfile
+from pathlib import Path
+
+import semver
+import yaml
+
+from artemis.module_base import ArtemisBase
+
+
+class BaseNewerVersionComparerModule(ArtemisBase):
+    software_name: str
+
+    def __init__(self, *args, **kwargs):  # type: ignore
+        super().__init__(*args, **kwargs)
+
+        self.endoflife_data_folder = tempfile.mkdtemp()
+        subprocess.call(
+            ["git", "clone", "https://github.com/endoflife-date/endoflife.date", self.endoflife_data_folder]
```

Comment:
I would put this as submodule, not explicitely cloning this everytime

Reply:
we do want to do this every container start so that we always have the latest version

Reply:
so put it in Dockerfile

Reply:
if we put it in Dockerfile, we would re-download this on all container **rebuilds**, not restarts - and currently the containers restart every 200 tasks, so e.g. Nuclei templates get redownloaded - I want the same behavior with endoflife